annotation/tests/test_tasks_services.py

```diff
@@ -295,3 +302,166 @@ def test_validate_user_actions_invalid_states(
             validation_user=True,
         )
     assert re.match(expected_error_message_pattern, excinfo.value.detail)
+
+
+def test_create_annotation_task():
+    with patch("sqlalchemy.orm.Session", spec=True) as mock_session:
+
+        db_session = mock_session()
+        create_annotation_task(
+            db_session,
+            ManualAnnotationTaskInSchema(
+                file_id=1,
+                pages={1, 2},
+                job_id=2,
+                user_id=uuid4(),
+                is_validation=True,
+                deadline=None,
+            ),
+        )
+
+        assert db_session.add.call_count == 2
+        db_session.commit.assert_called_once()
+
+
+@pytest.fixture
+def mock_db_session():
```

Comment:
you created mock db session fixture, but did not utilize in other tests, could you change it, please?

Reply:
Should I add this only for newly added tests in this pull request.

Reply:
for sure to tests in this branch, later you can replace the rest